### DISCUSSION TITLE
Trojan detected

### DISCUSSION THREAD

**Discussion by jon-riches** [geissbuehler/TermsrvPatcher#4] [General]
Version 1.1.0 is flagged as containing Trojan:Script/Oneeva.A!ml by Windows Defender

**Comment by geissbuehler** [comment]
I have reupped the zip archive in an uncompressed format, but with identical contents. It is very weird that the Defender complains about the zip file, but it is fine with the extracted files.

**Reply by jon-riches** [reply]
Great, thanks
